### ISSUE TITLE
Annotating database trigger.

### ISSUE THREAD

**Issue by bekicot** [ctran/annotate_models#918]
Is annotating database trigger is something that in scope with annotate models?

It would be nice to have them.

**Comment by gagalago** [comment]
agree, it will be nice. that will show these trigger that are "invisible" in the code to not forget their existence
